bot/admin/admin.py

```diff
@@ -133,6 +133,85 @@ async def embed_available_cogs(self, ctx: commands.Context):
 
         await ch_bot.send(embed=embed)
 
+    @cmd_for_bot_stuff.command(name='load', hidden=True)
+    @command_log
+    async def load_extension(self, ctx: commands.Context, extn_name: str):
+        """Command handler for the `bot` subcommand `load`.
+
+        Loads an extension (Cog) with the specified name into the bot.
+
+        Args:
+            ctx (discord.ext.commands.Context): The context from which this command is invoked.
+            extn_name (str): The name of the extension (Cog).
+        """
+        extn_name = extn_name.capitalize()
+
+        if "Cog" and "cog" not in extn_name:
+            extn_name += "Cog"
+        elif "cog" in extn_name:
+            extn_name = extn_name[:-3] + "Cog"
+
+        self.bot.load_extension(constants.INITIAL_EXTNS[extn_name])
+        print(f"\t* {extn_name} has been loaded.")
+
+    @cmd_for_bot_stuff.command(name='unload', hidden=True)
+    @command_log
+    async def unload_extension(self, ctx: commands.Context, extn_name: str):
+        """Command handler for the `bot` subcommand `unload`.
+
+        Removes an extension (Cog) with the specified name from the bot.
+
+        Args:
+            ctx (discord.ext.commands.Context): The context from which this command is invoked.
+            extn_name (str): The name of the extension (Cog).
+        """
+        extn_name = extn_name.capitalize()
+
+        if "Cog" and "cog" not in extn_name:
+            extn_name += "Cog"
+        elif "cog" in extn_name:
+            extn_name = extn_name[:-3] + "Cog"
+
+        self.bot.unload_extension(constants.INITIAL_EXTNS[extn_name])
+        print(f"\t* {extn_name} has been unloaded.")
+
+    @cmd_for_bot_stuff.group(name='reload', hidden=True, invoke_without_command=True)
+    @command_log
+    async def reload_extension(self, ctx: commands.Context, extn_name: str):
+        """Command handler for the `bot` subcommand `reload`.
+
+        Reloads an extension (Cog) with the specified name from the bot. If changes to the code inside a Cog have been
+        made, this is going to apply them without taking the bot offline.
+
+        Args:
+            ctx (discord.ext.commands.Context): The context from which this command is invoked.
+            extn_name (str): The name of the extension (Cog).
+        """
+        extn_name = extn_name.capitalize()
+
+        if "Cog" and "cog" not in extn_name:
```

Comment:
Same as above. 

Also third time copying code...

Reply:
![garbage](https://i.redd.it/hk54ti5n6tk11.png)

Reply:
But yeah, I was in a hurry. I'm gonna right a function for this. 😅